template/config/playwright/.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
```

Comment:
I feel that this file is unnecessary, because only a fraction of our users would use GitHub Actions.

Reply:
I might be opinionated since MS -> GitHub but I think that the vast majority is using GitHub Actions nowadays.
Anyway, I removed it for now to have it less opinionated.